js/event.ts

```diff
@@ -19,6 +23,7 @@ export class EventInit implements domTypes.EventInit {
 }
 
 export class Event implements domTypes.Event {
+  isTrusted!: boolean;
```

Comment:
I would suggest a comment, because unless someone introspects the constructor, they likely won't understand this.